project.godot

```diff
@@ -204,6 +204,11 @@ _global_script_classes=[ {
 "language": "GDScript",
 "path": "res://addons/escoria-core/game/scenes/dialogs/esc_dialog_player.gd"
 }, {
+"base": "ESCDialogManager",
```

Comment:
Like the last PR...is this ok to leave in? If so, no worries.

Reply:
Indeed, removing.